common/student/registration.ts

```diff
@@ -123,20 +121,18 @@ export async function becomeInstructor(student: Student, data: BecomeInstructorD
     }
```

Comment:
I think this can be removed for now, we do not want to invite instructors for screening anymore